VelocidiSDK/VelocidiSDKTests/RequestsTests.swift

```diff
@@ -0,0 +1,53 @@
+import UIKit
+import Quick
+import Nimble
+import Mockingjay
+import VelocidiSDK
+
+class MockUtil: VSDKUtil {
+    override func getAdvertisingIdentifier() -> UUID {
+        return UUID(uuidString: "00000000-0000-0000-0000-000000000000")!
+    }
+}
+
+class MockRequest: VSDKRequest<JSONModel> {
+    override init() {
+        super.init()
+        self.util = MockUtil()
+    }
+    
+}
+
+class RequestsTests: QuickSpec {
+    override func spec() {
+        super.spec()
+        describe("VSDKRequests") {
+            it("should not make a track requests if the advertisingIdentifier is all 0s") {
+                let url = "http://testdomain.com"
+                let trackingEvent = VSDKPageView()
+                trackingEvent.siteId = "0"
+                trackingEvent.clientId = "0"
+                
+                var requestExecuted: Bool = false
+
+                self.stub({(request: URLRequest) in
+                    return request.url!.absoluteString.starts(with: url)
+                }, failure(NSError(domain: url, code: 451)))
```

Comment:
What is the meaning of 451? Is the HTTP error code? Wouldn't it be better a [409](https://httpstatuses.com/409)?

Reply:
There isn't a specific motive, just thought it would be the most appropriate from all the 4XX error codes. 409 also does not seem like the best option (_conflict with the current state of the **target** resource_ is not the case here). Perhaps 403 is the best option!

Reply:
Sure, 403 seems fine 🙂

Reply:
Changed in 9579f1f